_pkgdown.yml

```diff
@@ -0,0 +1,13 @@
+authors:
```

Comment:
What is this file for?

Reply:
I think it is for automating package documentation.